lib/media/segment_reference.js

```diff
@@ -26,8 +26,10 @@ shaka.media.InitSegmentReference = class {
    * @param {?number} endByte The offset from the start of the resource to the
    *   end of the segment, inclusive.  A value of null indicates that the
    *   segment extends to the end of the resource.
+   * @param {?shaka.extern.MediaQualityInfo} mediaQuality Information about the
```

Comment:
For optional members, the normal syntax would be:
```
* @param {shaka.extern.MediaQualityInfo=}
```

Reply:
done